### DISCUSSION TITLE
Replicate all tables

### DISCUSSION THREAD

**Discussion by iberlob** [osalvador/ReplicaDB#84] [General]
Is there a way to replicate all the tables in a database?

I have a database with many tables to replicate to indicate them one by one

Thank you very much and great job

**Comment by osalvador** [comment]
Hi @iberlob ,

We are working on this feature, at the moment, you can create script for multiple table replication, see this comment: https://github.com/osalvador/ReplicaDB/issues/27#issuecomment-862229297 . Also you can use environment variables for replace the table names.
